stock_analysis_forecast/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Alex Comba - Agile Business Group
```

Comment:
As per https://github.com/OCA/stock-logistics-reporting/pull/27/files#diff-3f60dd96b612c425461d214c007a363bR52, should the all the copyright stuff belong to Odoo SA?